### PR TITLE
fix: v1 e2es

### PR DESCRIPTION
**Pull request by jamil314**
## Description

- Don't try to click Birth / Death certificate as the certificate template as it is pre selected.

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/testcases/print-certificate/birth/36.01-02-validation-of-printing-certificate.spec.ts [hunk 1]
```diff
@@ -29,39 +29,20 @@ test.describe.serial('Certified copies', () => {
...
-      await expect(
-        page.getByText('Please select certificate type')
-      ).toBeVisible()
       await expect(
         page.getByText('Please select who is collecting the certificate')
       ).toBeVisible()
     })
 
-    test('2.2 Click continue without selecting collector type', async () => {
+    test('2.2 Click continue without after selecting collector type', async () => {
```
Comment (Zangetsu101) [inline e2e/testcases/print-certificate/birth/36.01-02-validation-of-printing-certificate.spec.ts]:
```suggestion
    test('2.2 Click continue after selecting collector type', async () => {
```

**Review by Zangetsu101 — APPROVED** [review]
